conf.yaml

```diff
@@ -818,6 +818,18 @@ contents:
               -
                 repo:   ecctl
                 path:   docs/
+          - title:      Elastic Cloud Terraform Provider
+            prefix:     en/tpec
+            tags:       CloudTerraform/Reference
+            subject:    TPEC
+            current:    master
+            branches:   [ master ]
+            index:      docs-elastic/index.asciidoc
+            chunk:      1
+            sources:
+              -
+                repo:   terraform-provider-ec
```

Comment:
I think you need to add this at the top under `repos` as well. This is what's causing the build failure.

Reply:
Yup, good catch! I added the missing terraform-provider-ec repo at the top. Hopefully this will fix the failures.